src/po/cs.cp1250.po

```diff
@@ -13,7 +13,7 @@ msgstr ""
 "Last-Translator: Ji�� Pavlovsk� <jpavlovsky@mbox.vol.cz>\n"
 "Language-Team: Czech <cs@li.org>\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=cp1250\n"
+"Content-Type: text/plain; charset=CP1250\n"
```

Comment:
Note to reviewers: I have not touched any of the .po files, the changes were all auto-generated. I'm happy to change or revert anything in the .po files.

Reply:
you can reset that file (or copy it back from the main branch).